web_widget_float_formula/static/src/js/web_widget_float_formula.js

```diff
@@ -52,12 +53,7 @@ odoo.define('web_widget_float_formula', function(require) {
             },
 
             _process_formula: function(formula) {
-                try{
```

Comment:
Why is this catch being removed? This was a bug fix for an edge case #431 abfea0fff77466e268dfad8dd007a45c3ffbc0c0